app/models/confirmation.rb

```diff
@@ -1,6 +1,8 @@
 class Confirmation
   include ActiveModel::Model
-  attr_accessor :outcome, :message, :vo_number, :no_vo, :no_pvo, :renew_vo, :renew_pvo, :banned_visitors, :unlisted_visitors, :visitor_not_listed, :visitor_banned, :canned_response, :closed_visit
+  attr_accessor :outcome, :message, :vo_number, :no_vo, :no_pvo, :renew_vo,
+    :renew_pvo, :banned_visitors, :unlisted_visitors, :visitor_not_listed,
+    :visitor_banned, :canned_response, :closed_visit
```

Comment:
As you say, I think that would lead to a very long list at the top of the file, which might just be distracting. I don't think it's likely to be particularly volatile.